commands/deploy_command.go

```diff
@@ -652,28 +673,13 @@ func (deployCommandFlagsValidator) ValidateParsedFlags(flags *flag.FlagSet) erro
                 err = fmt.Errorf("%s is not a valid deployment strategy, available strategies: %v", f.Value.String(), AvailableStrategies())
                 return
             }
-        case timeoutOpt:
-            if e := ValidateTimeoutOption(f.Name, flags, 259200); e != nil {
-                err = e
-                return
-            }
-        case startTimeoutOpt:
-            if e := ValidateTimeoutOption(f.Name, flags, 259200); e != nil {
-                err = e
-                return
-            }
-        case stageTimeoutOpt:
+        case timeoutOpt, startTimeoutOpt, stageTimeoutOpt, uploadTimeoutOpt, taskExecutionTimeoutOpt:
```

Comment:
I think this was separate to allow different values for the validation 'maxAllowedVal'